x-pack/platform/plugins/shared/ml/public/application/model_management/deployment_setup.tsx

```diff
@@ -274,7 +274,7 @@ export const DeploymentSetup: FC<DeploymentSetupProps> = ({
               'xpack.ml.trainedModels.modelsList.startDeployment.cloudAutoscaling.lowCpuStaticHelp',
               {
                 defaultMessage:
-                  'This level sets resources to the minimum required for ELSER to run if supported by your Cloud console selection. It may not be sufficient for a production application.',
+                  'This level sets resources to the minimum required for model to run if supported by your Cloud console selection. It may not be sufficient for a production application.',
```

Comment:
nit - `the model to run...`

Reply:
updated 9cf5d78e05caea551443efe450c5f137e38d7be9